great_expectations/expectations/core/expect_table_row_count_to_be_between.py

```diff
@@ -71,14 +72,64 @@ class ExpectTableRowCountToBeBetween(TableExpectation):
     success_keys = (
         "min_value",
         "max_value",
+        "auto",
+        "profiler_config",
     )
+
+    default_profiler_config: RuleBasedProfilerConfig = RuleBasedProfilerConfig(
+        name="expect_table_row_count_to_be_between",  # Convention: use "expectation_type" as profiler name.
+        config_version=1.0,
+        variables={
+            "num_bootstrap_samples": 9999,
+            "false_positive_rate": 0.05,
+            "truncate_values": {
+                "lower_bound": 0,
+                "upper_bound": None,
+            },
+            "round_decimals": 0,
+        },
+        rules={
+            "row_count_range_rule": {
```

Comment:
can we type this as well?

Reply:
@donaldheppner Not yet -- for now, we follow exactly the same pattern for "Base" level business objects' configuration objects as established in `DataContextConfig` and `CheckpointConfig`.  Potentially can be done in the future, but a bit too many changes if attempted to do here.

Reply:
@donaldheppner Also, we are keeping the option for users to specify either dictionary or config object in the calls to `validator.expect_*`.  If/when the product decision is made to remove this flexibility, then it would be a very quick fix.  Thanks.